window.go

```
@@ -1452,22 +1452,24 @@ func (wb *WindowBase) Visible() bool {
 
 // SetVisible sets if the *WindowBase is visible.
 func (wb *WindowBase) SetVisible(visible bool) {
-	old := wb.Visible()
+	if wb.Visible() != visible {
+		setWindowVisible(wb.hWnd, visible)
+	}
+}
 
-	setWindowVisible(wb.hWnd, visible)
+func (wb *WindowBase) updateVisibility(visible bool) {
+	old := wb.Visible()
+	if old != visible {
+		return
+	}
```

Why do we need this? From the `(*WindowBase).updateVisibility` usage, it appears that `old != visible` can never be true. And if the condition were ever true, that would indicate a coding error / incorrect usage of the method.

So, maybe remove it completely, or update it to panic instead of returning?
And if you choose to keep it, could you change the condition to:
```go
if wb.Visible() != visible {
```
?

---

I'll remove it. Thanks!